atomicfu/build.gradle

```diff
@@ -17,6 +19,10 @@ ext {
     }
 }
 
+// TODO: this block should be removed after the switch to Kotlin language version 1.9
```

Comment:
If we settle on KGP version check in the end, corresponding TODOs (three of them in `gradle.build`) should probably also be updated (changed from "Kotlin language version 1.9" to "Kotlin release version 1.9.0" or smth) to escape potential future confusion when migrating to KGP 1.9.0 even without changing the language version.